stellar/deploy-contract.js

```diff
@@ -109,9 +131,13 @@ function main() {
     addBaseOptions(program, { address: true });
 
     program.addOption(new Option('--initialize', 'initialize the contract'));
-    program.addOption(new Option('--contract-name <contractName>', 'contract name to deploy').makeOptionMandatory(true));
-    program.addOption(new Option('--wasm-path <wasmPath>', 'path to the WASM file').makeOptionMandatory(true));
+    program.addOption(new Option('--contract-name <contractName>', 'contract name to deploy'));
+    program.addOption(new Option('--wasm-path <wasmPath>', 'path to the WASM file'));
     program.addOption(new Option('--nonce <nonce>', 'optional nonce for the signer set'));
+    program.addOption(new Option('--install', 'install only'));
+    program.addOption(new Option('--upgrade', 'upgrade only'));
```

Comment:
create separate `deploy`/`upgrade` cmds. See CLI setup in `sui/deploy-contract.js`

Reply:
got it. updated.